sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -1432,4 +1432,15 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
       getMessage()
     assert(e1.startsWith("Path does not exist"))
   }
+
+  test("test !< and !> comparator") {
```

Comment:
Hi @rxin, 
Thanks for your comment! I've removed this test case from DataFrameSuite. Please free free to let me know of any other comments or questions.